quixstreams/platforms/quix/config.py

```diff
@@ -360,8 +359,19 @@ def _finalize_create(self, topics: Set[str], timeout: Optional[int] = None):
                 if topic["status"] == "Ready":
                     logger.debug(f"Topic {topic['name']} creation finalized")
                     topics.remove(topic["id"])
+                elif topic["status"] == "Error":
+                    logger.debug(f"Topic {topic['name']} encountered an error")
+                    exceptions[topic["name"]] = (
+                        {
+                            "current_error": topic["errorStatus"],
+                            "previous_error": topic["lastError"],
+                        },
+                    )
+                    topics.remove(topic["id"])
```

Comment:
Also, I think the semantics of "current_error" and "previous_error" are not exactly correct here.

According to the [Swagger](https://portal-api.platform.quix.ai/swagger/index.html?urls.primaryName=Portal%20API%20V1) doc, the `"errorStatus"` field seems to indicate the stage of the topic lifecycle where the error happened.  
And the `"lastError"` field contains the error description.

Reply:
I hear you on not complicating it, but I figured we should be consistent with how non-quix `create_topics` works.

Also, it might be useful when you've maybe somehow messed up multiple topics so you dont have to debug them one by one (which is also why I had the non-quix `create_topics` collects the errors too). 

Granted, I don't know if errors really manifest in the same way there, plus Quix topics do seem to take much longer to finalize.

Reply:
> I hear you on not complicating it, but I figured we should be consistent with how non-quix create_topics works.

Oh, ok, I see the logic now.

Let's keep this part consistent then